webdriver/tests/bidi/script/call_function/strict_mode.py

```diff
@@ -25,7 +25,8 @@ async def test_strict_mode(bidi_session, top_context):
     )
     assert result == {
         "type": "number",
-        "value": 1}
+        "value": 1
```

Comment:
```suggestion
        "value": 1,
```

Shouldn't this get a final comma as well? Or does only ESLint enforce it? This would apply to other changes as well.